fiware_cloto/cloto/manager/InfoManager.py

```diff
@@ -89,3 +96,46 @@ def init_information(self):
         s = ServerInfo(id=1, owner=settings.OWNER, version=settings.VERSION,
                        runningfrom=runningfrom, doc=settings.API_INFO_URL)
         s.save()
+
+    def publish_message(self, message):
+        """Publish a message related to the windowsize in the rabbitmq and
+        close the connection
+
+        :param str message:       The well-formatted message to send
+        """
+        """ Initialize the class and create a connection with a RabbitMQ server instance.
+        """
+        # Get the default IP address of the RabbitMQ server.
+        rabbitmq = settings.RABBITMQ_URL
+        self.connection = None
+        self.channel = None
+        # Open a remote connection to RabbitMQ on specified IP address
+        try:
+            self.connection = pika.BlockingConnection(pika.ConnectionParameters(
+                host=rabbitmq))
+
+            # Open the channel
+            self.channel = self.connection.channel()
+        except (pika.exceptions.AMQPConnectionError, pika.exceptions.AMQPChannelError), error:
+            raise Exception("Error with Rabbit connection %s", error)
+
+        if self.channel:
+            self.channel.exchange_declare(exchange="windowsizes",
```

Comment:
Do you need _declare_ the exchange? What happens if facts has already declared the exchange?